set.mm

```diff
@@ -637098,6 +637098,72 @@ deduction version (Contributed by metakunt, 22-May-2024.) $)
+    intlewftc.9 $e |- ( ph -> E e. ( ( A (,) B ) -cn-> RR ) ) $.
+    intlewftc.10 $e |- ( ph -> D e. L^1 ) $.
+    intlewftc.11 $e |- ( ph -> E e. L^1 ) $.
+    intlewftc.12 $e |- ( ph -> D = ( x e. ( A (,) B ) |-> P ) ) $.
+    intlewftc.13 $e |- ( ph -> E = ( x e. ( A (,) B ) |-> Q ) ) $.
+    intlewftc.14 $e |- ( ( ph /\ x e. ( A (,) B ) ) -> P <_ Q ) $.
+    intlewftc.15 $e |- ( ph -> ( F ` A ) <_ ( G ` A ) ) $.
+    $( Inequality inference by invoking fundamental theorem of calculus.
+       (Contributed by metakunt, 22-Jul-2024.) $)
+    intlewftc $p |- ( ph -> ( F ` B ) <_ ( G ` B ) ) $=
+      ( vt cfv cmin co caddc cle wbr cicc cr ccncf wcel wf cncff syl leidd 3jca
+      w3a wb elicc2 syl2anc mpbird ffvelrnd resubcld cioo citg cibl cmpt eleq1d
+      mpbid feq1d fvmptelrn itgle itgmpt wceq fveq1d adantr eqcomd itgeq2dv cdv
+      cv wa cc wss ax-resscn a1i fss ssidd cncffvrn eqeltrrd ftc2 eqtrd breq12d
+      le2addd sseldi npcand ) ADIUGZCIUGZUHUIZXBUJUIZDJUGZCJUGZUHUIZXFUJUIZUKUL
+      XAXEUKULAXCXBXGXFAXAXBACDUMUIZUNDIAIXIUNUOUIZUPZXIUNIUQZNXIUNIURUSZADXIUP
+      ZDUNUPZCDUKULZDDUKULZVBZAXOXPXQLMADLUTVAACUNUPZXOXNXRVCKLCDDVDVEVFZVGZAXI
+      UNCIXMACXIUPZXSCCUKULZXPVBZAXSYCXPKACKUTMVAAXSXOYBYDVCKLCDCVDVEVFZVGZVHYF
+      AXEXFAXIUNDJAJXJUPZXIUNJUQZOXIUNJURUSZXTVGZAXIUNCJYIYEVGZVHYKABCDVIUIZFVJ
+      ZBYLGVJZUKULXCXGUKULABYLFGAEVKUPBYLFVLZVKUPTAEYOVKUBVMVNAHVKUPBYLGVLZVKUP
+      UAAHYPVKUCVMVNABYLFUNAYLUNEUQZYLUNYOUQAEYLUNUOUIZUPZYQRYLUNEURUSZAYLUNEYO
+      UBVOVNVPZABYLGUNAYLUNHUQZYLUNYPUQAHYRUPZUUBSYLUNHURUSZAYLUNHYPUCVOVNVPZUD
+      VQAYMXCYNXGUKAYMUFYLUFWEZYOUGZVJZXCABUFYLFUNUUAVRAUUHUFYLUUFEUGZVJZXCAUFY
+      LUUGUUIAUUFYLUPZWFZUUIUUGAUUIUUGVSUUKAUUFEYOUBVTWAWBWCAUUJUFYLUUFUNIWDUIZ
+      UGZVJXCAUFYLUUIUUNUULUUFEUUMAEUUMVSUUKPWAVTWCAUFCDIKLMAEYLWGUOUIZUPZUUMUU
+      OUPAUUPYLWGEUQZAYQUNWGWHZUUQYTUURAWIWJZYLUNWGEWKVEAWGWGWHZYSUUPUUQVCAWGWL
+      ZRYLUNWGEWMVEVFAEUUMUUOPVMVNAEUUMVKPTWNAIXIWGUOUIZUPZXIWGIUQZAXLUURUVDXMU
+      USXIUNWGIWKVEAUUTXKUVCUVDVCUVANXIUNWGIWMVEVFWOWPWPWPAYNUFYLUUFHUGZVJZXGAY
+      NUFYLUUFYPUGZVJUVFABUFYLGUNUUEVRAUFYLUVGUVEUULUUFYPHUULHYPAHYPVSUUKUCWAWB
+      VTWCWPAUVFUFYLUUFUNJWDUIZUGZVJXGAUFYLUVEUVIUULUUFHUVHAHUVHVSUUKQWAVTWCAUF
+      CDJKLMAHUUOUPZUVHUUOUPZAUVJUVKAUVJUVKAUVJYLWGHUQZAUUBUURUVLUUDUUSYLUNWGHW
+      KVEAUUTUUCUVJUVLVCUVASYLUNWGHWMVEVFAHUVHUUOQVMZVNUVMVFUVMVNAHUVHVKQUAWNAJ
+      UVBUPZXIWGJUQZAYHUURUVOYIUUSXIUNWGJWKVEAUUTYGUVNUVOVCUVAOXIUNWGJWMVEVFWOW
+      PWPWQVNUEWRAXDXAXHXEUKAXAXBAUNWGXAWIYAWSAUNWGXBWIYFWSWTAXEXFAUNWGXEWIYJWS
+      AUNWGXFWIYKWSWTWQVN $.
+  $}
+
+  ${
+    $d A k $.  $d N k $.  $d k ph $.
+    aks4d1p1p1.1 $e |- ( ph -> A e. RR+ ) $.
+    aks4d1p1p1.2 $e |- ( ph -> N e. NN ) $.
+    $( Exponential law for finite products, special case.  (Contributed by
+       metakunt, 22-Jul-2024.) $)
+    aks4d1p1p1 $p |- ( ph -> prod_ k e. ( 1 ... N ) ( A ^c k ) =
+     ( A ^c sum_ k e. ( 1 ... N ) k ) ) $=
```

Comment:
I believe this time, this is generic enough to get a name without the `aks`, but I'm unsure what name to suggest.

Reply:
Yeah, if someone needs it and it is moved outside of the mathbox. I agree.